src/Microsoft.Dnx.Tooling/Building/BuildManager.cs

```diff
@@ -296,6 +310,97 @@ private bool BuildInternal(string projectPath)
             return success;
         }
 
+        private void AddPackageFiles(string projectDirectory, IEnumerable<PackIncludeEntry> packageFiles, PackageBuilder packageBuilder, IList<DiagnosticMessage> diagnostics)
+        {
+            var rootDirectory = new DirectoryInfoWrapper(new DirectoryInfo(projectDirectory));
+
+            foreach (var match in CollectAdditionalFiles(rootDirectory, packageFiles, _currentProject.ProjectFilePath, diagnostics))
+            {
+                packageBuilder.Files.Add(match);
+            }
+        }
+
+        internal static IEnumerable<PhysicalPackageFile> CollectAdditionalFiles(DirectoryInfoBase rootDirectory, IEnumerable<PackIncludeEntry> projectFileGlobs, string projectFilePath, IList<DiagnosticMessage> diagnostics)
+        {
+            foreach (var entry in projectFileGlobs)
+            {
+                // Evaluate the globs on the right
+                var matcher = new Matcher();
+                matcher.AddIncludePatterns(entry.SourceGlobs);
+                var results = matcher.Execute(rootDirectory);
+                var files = results.Files.ToList();
+
+                // Check for illegal characters
+                if(string.IsNullOrEmpty(entry.Target))
+                {
+                    diagnostics.Add(new DiagnosticMessage(
+                        $"Invalid '{Runtime.Project.PackIncludePropertyName}' section. The target '{entry.Target}' is invalid, " +
+                        "targets must either be a file name or a directory suffixed with '/'. " +
+                        "The root directory of the package can be specified by using a single '/' character.",
+                        projectFilePath,
+                        DiagnosticMessageSeverity.Error,
+                        entry.Line,
+                        entry.Column));
+                    continue;
+                }
+
+                if (entry.Target.Split('/').Any(s => s.Equals(".") || s.Equals("..")))
```

Comment:
"rooted" paths in the target refer to the root of the package, so no. See the error message above